src/common/pure/CurrencyLogo/hooks/useCurrencyLogoURIs.ts

```diff
@@ -42,9 +42,9 @@ function getTokenLogoURI(address: string, chainId: SupportedChainId = SupportedC
 const currencyLogoCache = new Map<string, Array<string>>()
 // TODO: must be refactored
 export default function useCurrencyLogoURIs(currency?: Currency | null): string[] {
-  const externalLogo = useExternalTokenLogo(currency)
-  // There is a modification of Token in useDetectNativeToken()
   const currencyAddress = currency ? (currency.isNative ? NATIVE_CURRENCY_BUY_ADDRESS : currency.address) : null
+  const externalLogo = useProxyTokenLogo(currency?.chainId, currencyAddress)
+  // There is a modification of Token in useDetectNativeToken()
```

Comment:
Is this comment relevant to your line 46?

Reply:
Fixed